### PR TITLE
docs: add getting started page

### PR DESCRIPTION
**Pull request by SahilAujla**
## PR Overview
- Adds a "Getting Started" page with code snippet for sending userops through Light Account.
- Re-order the pages on the left to move "Packages Overview" towards the end, after "Using Smart Accounts" section.

## Open Questions
- Should we provide a code snippet with actual values for `target`, `data` and `value` to send a user operation? Instead of leaving them with placeholders for user to add?
- If so, which factory address should we use for LightAccount? (as it's not final yet?)
- Currently the guide just gives the code snippet for a user to use LightAccount and send user ops, however, they would need to have some ETH in their smart account for this to work.
- Should we add a code snippet for determining the counterfactual address and asking them to send ETH to that address? Though, this comes at the cost of taking the simplicity of the guide away upto some extent and adds some resistance.
- Or should we use the Gas Manager in the same guide so they won't have to even have ETH in their account and can run it without any resistance? But again this comes at the cost of taking away simplicity of the guide away upto some extent as in this case, we would have to send a user operation that does some function interaction on a smart contract (instead of just a simple ETH transfer)

### CODE REVIEW

File: site/.vitepress/config.ts [hunk 1]
```diff
@@ -56,6 +52,10 @@ export default defineConfig({
           { text: "Transferring Ownership", link: "/transferring-ownership" },
         ],
       },
+      {
+        text: "Packages Overview",
+        link: "/packages/overview",
+      },
```
Comment (moldy530) [inline site/.vitepress/config.ts:58]: nit: can we leave this in the top section. I initially had it as you had it and it looked a bit weird styling-wise, but happy to revisit this

File: site/getting-started.md [hunk 2]
```diff
@@ -1,7 +1,74 @@
...
+    - property: og:description
+      content: Getting Started with Alchemy's Account Kit
 ---
 
 # Getting Started
 
-TODO
+This guide will help you get started with Account Kit by setting up your environment, creating a Light Account (a type of smart account implementation), and sending a User Operation from it. By the end of this guide, you'll have a basic understanding of how to use the SDK and where to look for more advanced use cases.
+
+### Install the Packages
```
Comment (moldy530) [inline site/getting-started.md:19]: nit:
```suggestion
## Install the Packages
```

File: site/getting-started.md [hunk 3]
```diff
@@ -1,7 +1,74 @@
...
+npm install @alchemy/aa-alchemy @alchemy/aa-accounts @alchemy/aa-core viem
+```
+
+```bash [yarn]
+yarn add @alchemy/aa-alchemy @alchemy/aa-accounts @alchemy/aa-core viem
+```
+
+:::
+
+### A Simple Light Account Example
```
Comment (moldy530) [inline site/getting-started.md:52]:
```suggestion
## A Simple Light Account Example
```

File: site/getting-started.md [hunk 4]
```diff
@@ -1,7 +1,74 @@
...
+
+::: tip Note
+Remember to:
+
+1. Replace `"0xYourEOAPrivateKey"` with your actual EOA private key.
+2. Set `"ALCHEMY_API_KEY"` with your unique Alchemy API key.
+3. Adjust the `target` and `data` fields in the `sendUserOperation` function to match your requirements.
+   :::
+
+### Dive Deeper
```
Comment (moldy530) [inline site/getting-started.md:68]:
```suggestion
## Dive Deeper
```

File: site/getting-started.md [hunk 5]
```diff
@@ -1,7 +1,74 @@
...
+
+In this guide we initialized `provider` with the `aa-alchemy` package however we could have done the same with the other packages of Account Kit as well. To learn more about the different packages and their use cases, check out the [packages overview](/packages/overview.html) page.
+
+### Next Steps
```
Comment (moldy530) [inline site/getting-started.md:72]:
```suggestion
## Next Steps
```

File: site/getting-started.md [hunk 6]
```diff
@@ -1,7 +1,74 @@
...
+Remember to:
+
+1. Replace `"0xYourEOAPrivateKey"` with your actual EOA private key.
+2. Set `"ALCHEMY_API_KEY"` with your unique Alchemy API key.
+3. Adjust the `target` and `data` fields in the `sendUserOperation` function to match your requirements.
+   :::
+
+### Dive Deeper
+
+In this guide we initialized `provider` with the `aa-alchemy` package however we could have done the same with the other packages of Account Kit as well. To learn more about the different packages and their use cases, check out the [packages overview](/packages/overview.html) page.
```
Comment (moldy530) [inline site/getting-started.md:70]:
```suggestion
In this guide we initialized `provider` with the `aa-alchemy` package however we could have done the same with the other packages of Account Kit as well. To learn more about the different packages and their use cases, check out the ["Packages Overview"](/packages/overview) page.
```

**Review by moldy530 — COMMENTED** [review]
Couple nits, but LGTM otherwise

re: user's need to have ETH to send a user operation, we should either:
1. Add a ::: tip explaining they need to fund their address
2. (preferred) under the `const provider` definition in the snippet, add a `console.log(await provider.getAddress())` and have a comment there about needing to fund that address with ETH if they want to be able to send Tx's successfully

File: site/.vitepress/config.ts (hunk 1 above)
Comment (SahilAujla) [reply site/.vitepress/config.ts]: reverted

**Comment by SahilAujla** [comment]
> Couple nits, but LGTM otherwise
> 
> re: user's need to have ETH to send a user operation, we should either:
> 
> 1. Add a ::: tip explaining they need to fund their address
> 2. (preferred) under the `const provider` definition in the snippet, add a `console.log(await provider.getAddress())` and have a comment there about needing to fund that address with ETH if they want to be able to send Tx's successfully

Done!

**Pull request merged**